e2e/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "e2e",
+  "version": "1.0.0",
+  "scripts": {
+    "e2e-setup": "npx playwright install",
+    "e2e-tests": "dotenv -- cross-var npx playwright test --config %APP_NAME%/playwright.config.js",
+    "e2e-tests-ci": "cross-var npx playwright test --config %APP_NAME%/playwright.config.js",
```

Comment:
Not sure if there's one command for both `e2e-tests` and `e2e-tests-ci`

Reply:
`e2e-test` vs `e2e-tests` plural?

Reply:
`e2e-test` to be consistent with `npm test`, `make test`, etc